lib/src/base-styles/_nx-tile.scss

```diff
@@ -9,6 +9,10 @@ $nx-tile-header-height: 24px; // Minimum height
 $nx-tile-header-height-actions: 43px; // Height with sub-title and/or right actions
 $nx-tile-background-color: $nx-white;
 
+// the "padding" around the inside edge of the nx-tile, which must be implemented using padding/margin
+// on children since nx-tile-content--accordion-container is supposed to go all the way to the edges
+$nx-tile-padding: $nx-spacing-l;
```

Comment:
You've created this variable, but I can't see anywhere that you have used it.

Reply:
> You've created this variable, but I can't see anywhere that you have used it.

Whoops. e6733595